server/publicapi/features/returning_errors.feature

```diff
@@ -2,16 +2,29 @@ Feature: Returning error on invalid request parameters
 
     Scenario: Sending an unknown parameter when retrieving items
         When we get "/items?parameter_x=foo"
-        Then we get error 422
+        Then we get error 10001
```

Comment:
I think I prefer to be consistent with how SD API treats errors, this is something we can discuss, having standalone error codes looks better to me.